nixos/modules/services/networking/networkmanager.nix

```diff
@@ -387,7 +409,8 @@ in {
       [ "networking" "networkmanager" "packages" ]
       [ "networking" "networkmanager" "plugins" ])
     (mkRenamedOptionModule [ "networking" "networkmanager" "useDnsmasq" ] [ "networking" "networkmanager" "dns" ])
-    (mkRemovedOptionModule ["networking" "networkmanager" "dynamicHosts"] ''
+    (mkRenamedOptionModule [ "networking" "networkmanager" "enableFccUnlock" ] [ "networking" "networkmanager" "enableBundledFccUnlockScripts" ])
+    (mkRemovedOptionModule [ "networking" "networkmanager" "dynamicHosts" ] ''
```

Comment:
Could you please extract the cleanups to a separate commit?

Reply:
I can split the rename into a commit before the actual change, sure.

Reply:
pushed.